contents/docs/cloud/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: PostHog Cloud
+sidebarTitle: Overview
+sidebar: Docs
+showTitle: true
+---
+
+PostHog Cloud is our hosted and managed version of the PostHog open-source platform. It always runs on the latest release with updates and migrations managed for you.
+
```

Comment:
maybe add a big sign-up button here for the impatient folks

Reply:
Interesting idea. @corywatilo what do you think about adding larger CTAs to documentation content?

My $0.02 is that although docs do have a marketing role to play, they shouldn't have blatant marketing content such as CTA buttons in the main content when links are sufficient.

Reply:
Flyby comment: I think it's generally OK to have big CTAs at the _bottom_ of a blog post — I've seen that pattern a lot, whether the CTA is "Donate to this cause," "Subscribe," "Click this Amazon link," et cetera.

Generally it won't interrupt the reading flow, and only the most committed readers who have made it to the bottom of the page will follow through.

Reply:
Agree on the blog post. I feel that's more marketing-y. Less so in the middle of docs. I'm not opposed to a CTA button though if others feel strongly about it.